CMEW/meta/rose-meta.conf

```
@@ -115,22 +115,56 @@ type=integer
 
 [template variables=MODEL_ID]
 compulsory=true
-description=A short name for the model simulation.
+description=A short name for the run to be evaluated.
 help=This name is included in file paths so must only contain letters, numbers,
     =underscores or dashes. Therefore spaces, '#' and other special characters are
     =not allowed.
 pattern=^"[a-zA-Z0-9_-]+"$
-sort-key=2
+sort-key=02
 type=quoted
 
 [template variables=NUMBER_OF_YEARS]
 compulsory=true
 description=The number of years of model data to evaluate.
 help=Data evaluated will end on 31 December START_YEAR + NUMBER_OF_YEARS -1,
     =inclusive.
-sort-key=4
+sort-key=22
 type=integer
 
+[template variables=REF_CALENDAR]
+compulsory=true
+description=The calendar to be used as reference.
+sort-key=14
+values="gregorian","360_day"
+
+[template variables=REF_MODEL_ID]
+compulsory=true
+description=A short name for the run to be used as reference.
+help=This name is included in file paths so must only contain letters, numbers,
+    =underscores or dashes. Therefore spaces, '#' and other special characters are
+    =not allowed.
+pattern=^"[a-zA-Z0-9_-]+"$
+sort-key=12
+type=quoted
+
+[template variables=REF_SUITE_ID]
+compulsory=true
+description=The ID of the model simulation suite to be used as reference.
+help=Must only contain letters, numbers, underscores or dashes.
+pattern=^"[a-zA-Z0-9_-]+"$
+sort-key=11
+type=quoted
+
+[template variables=REF_VARIANT_LABEL]
+compulsory=true
+description=Ensemble member variant label for the reference run.
+help=This name is included in file paths so must only contain letters, numbers,
+    =underscores or dashes. Therefore spaces, '#' and other special characters are
+    =not allowed.
+pattern=^"[a-zA-Z0-9_-]+"$
```

```suggestion
help=Must adhere to CMIP6 variant label format: r<int>i<int>p<int>f<int>.
    =E.g. r2i1p1f3. https://help.ceda.ac.uk/article/4801-cmip6-data
pattern=^"r[0-9]+i[0-9]+p[0-9]+f[0-9]+"$
```
I just remembered that variant label has a different constraint from the other variables.